cores/migrations/20240725020036_MakeSummaryNullable.up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE markdowns
+AlTER COLUMN summary
+DROP NOT NULL;
+
+ALTER TABLE markdowns
+AlTER COLUMN summary
+DROP DEFAULT;
```

Comment:
Summary was a non-null column with an empty string default

But we make a markdowns before we have a summary, so it make more sense for it to be nullable and not have a default empty string value